_15thnight/models.py

```diff
@@ -1,10 +1,11 @@
 """Data Models."""
-import uuid
 from datetime import datetime, timedelta
+import uuid
```

Comment:
I typically sort by module name being imported. In this case, uuid comes after datetime alphabetically. I also group, in order, stdlib imports, 3rd party imports, project imports.